lib/Service/TokenService.php

```diff
@@ -177,6 +181,7 @@ public function refresh(Token $token): Token {
 						'client_secret' => $clientSecret,
 						'grant_type' => 'refresh_token',
 						'refresh_token' => $token->getRefreshToken(),
+						'scope' => $scope,
```

Comment:
this not working. maybe we cannot request new scopes while refreshing token :thinking:

Reply:
We will hit a blockage with `ExternalTokenRequestedEvent` because the login token doesn't contain the extra scope (e.g.: `api_v3`) and we cannot request new scopes while refreshing the token.

Reply:
It seems expected that the refreshed token has the same scopes.
In this case the scopes must be requested in the provider configuration of user_oidc so that the login token itself already has those scopes.